controllers/runner_manager.go

```diff
@@ -477,17 +477,17 @@ func (p *manageProcess) maintainRunnerPods(ctx context.Context, runnerList []*gi
 						return ctrl.SetControllerReference(po, pdb, p.scheme)
 					})
 					if err != nil {
-						log.Error(err, "failed to create or update protection pdb: %s", po.Name)
+						log.Error(err, "failed to create or update protection pdb", "pdb", po.Name)
 						continue
 					}
-					log.Info("created or updated protection pdb: %s", po.Name)
+					log.Info("created or updated protection pdb", "pdb", po.Name)
 					po.Labels[constants.RunnerPodName] = po.Name
 					err = p.k8sClient.Update(ctx, po)
 					if err != nil {
-						log.Error(err, "failed to relabel runner pod: %s", po.Name)
+						log.Error(err, "failed to relabel runner pod", "pod", po.Name)
```

Comment:
The pod name is set here. So we need not set it :)
https://github.com/cybozu-go/meows/blob/v0.8.0/controllers/runner_manager.go#L378

Reply:
oh